app/models/plan.rb

```diff
@@ -34,6 +34,19 @@ class Plan < ApplicationRecord
   validates :name, presence: true
   validates :term, inclusion: TERM_TYPES
 
+  def attributes
+    {
+      id: nil,
+      name: nil,
+      term: nil,
+      assessment_id: nil,
+      disease_ids: nil,
+      user_id: nil,
+      created_at: nil,
+      updated_at: nil,
```

Comment:
I'm thinking that these fields are not needed here: `assessment_id`, `created_at`, `updated_at`, and `user_id`. My thinking is that we should only output what's actually needed/used on the client.

Reply:
removed unnecessary fields